server/applicationPattern/applicationPattern/onfModel/models/layerProtocols/TcpServerInterface.js

```diff
@@ -115,5 +114,97 @@ class TcpServerInterface extends layerProtocol {
     }
+
+    /**
+     * @description This function modifies the tcp-server local-address.
+     * @param {String} tcpServerUuid : tcp-server uuid to set tcp-server instance.
+     * @param {String} localAddress : localAddress that needs to be modified.
+     * @returns {promise} boolean {true|false}
+     **/
+    static setLocalAddressAsync(tcpServerUuid, localAddress) {
+        return new Promise(async function (resolve, reject) {
+            let isUpdated = false;
+            try {
+                let addressToBeDeleted = await fileOperation.readFromDatabaseAsync(
+                    onfPaths.TCP_CLIENT_LOCAL_ADDRESS.replace(
+                        "{uuid}", tcpServerUuid)
+                );
+                let addressPaths = await getPaths(tcpServerUuid, localAddress, addressToBeDeleted);
+                let localAddressPath = addressPaths[0];
+                let pathToBeDeleted = addressPaths[1];
+                if (pathToBeDeleted != undefined) {
+                    await fileOperation.deletefromDatabaseAsync(
+                        pathToBeDeleted,
+                        addressToBeDeleted,
+                        false
+                    );
+                }
+                isUpdated = await fileOperation.writeToDatabaseAsync(
+                    localAddressPath,
+                    localAddress,
+                    false);
+                resolve(isUpdated);
+            } catch (error) {
+                reject(error);
+            }
+        });
+    }
+
 }
+
+/**
+ * @description This function returns the local address configured .
+ * @param {String} localAddress : local address of the tcp server .
+ * @returns {promise} string {undefined | address}
+ **/
+function getConfiguredLocalAddress(localAddress) {
+    return new Promise(async function (resolve, reject) {
+        let address = {};
+        try {
+            if (!("domain-name" in localAddress)) {
+                address["ip-address"] = localAddress;
+            } else {
+                address = localAddress;
+            }
+            resolve(address);
+        } catch (error) {
+            reject(error);
+        }
+    });
+}
+
+/**
+ * @description This function returns the remote address configured .
+ * @param {String} tcpClientUuid : tcp-client uuid of the tcp-client instance
+ * @param {String} addressToBeDeleted : tcp-client address to be deleted.
+ * @param {String} remoteAddress : remote address of the tcp client .
+ * @returns {promise} list {paths}
+ **/
+function getPaths(tcpServerUuid, localAddress, addressToBeDeleted) {
+    return new Promise(async function (resolve, reject) {
+        let paths = [];
+        let localAddressPath;
+        let pathOfAddressToBeDeleted;
+        try {
+            if ("domain-name" in localAddress) {
+                localAddressPath = onfPaths.TCP_SERVER_DOMAIN_NAME.replace(
+                    "{uuid}", tcpServerUuid);
+                if (!("domain-name" in addressToBeDeleted))
```

Comment:
Please remove the hardcoded values. Instead , declare them as a constant in the module applicationPattern\onfModel\constants\OnfAttributes.js and utilize the declared constant. In this way , we can utilize the attribute name globally across many modules.